### PR TITLE
.coafile: Added .md file extension

### PR DESCRIPTION
**Pull request by SerekKiri**
This commit add markdown files extension to whitespace consistency rule

Closed issue: https://github.com/coala/projects/issues/529

### CODE REVIEW

**Comment by stellargo** [comment]
Correct typo ussles --> useless
Add --> Adds
files --> file
Add full stop at the end :)

File: _projects/extend-linter-integration.md [hunk 1]
```diff
@@ -93,12 +93,12 @@ difficult to debug things like:
 E           AssertionError: 0 == 0 : The local bear 'CheckstyleBear' yields no result although it should.
 E           Running bear CheckstyleBear...
 E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/
-			.local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
-			CheckstyleGood.java'
+            .local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
```
Comment (jayvdb) [inline _projects/extend-linter-integration.md]: indent this line, and the next one, with a few extra spaces (4 extra spaces?)

File: _projects/extend-linter-integration.md [hunk 2]
```diff
@@ -93,12 +93,12 @@ difficult to debug things like:
...
+            CheckstyleGood.java'
 E           Running bear CheckstyleBear...
 E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar
-			-c /home/ubuntu/.local/share/coala-bears/CheckstyleBear/google.xml
-			/home/ubuntu/coala-bears/tests/java/test_files/CheckstyleBad.java'
+            -c /home/ubuntu/.local/share/coala-bears/CheckstyleBear/google.xml
```
Comment (jayvdb) [inline _projects/extend-linter-integration.md]: again add some more spaces to this line, and the next one.

File: _faq/gsoc-requirements.md [hunk 3]
```diff
@@ -5,5 +5,6 @@ Other than the [requirements from Google](https://developers.google.com/open-sou
 
 * You have completed the [coala Newcomers guide](https://coala.io/newcomer)
 * You have a submission to [our usability survey](https://coala.io/usability)
-	so we know that you know how to *use coala*.
-	
\ No newline at end of file
+    so we know that
```
Comment (jayvdb) [inline _faq/gsoc-requirements.md]: "so" should be immediately under "You"

File: _faq/is-mentor-necessary.md [hunk 4]
```diff
@@ -1,6 +1,5 @@
 ---
-question: "A showcased project on the website doesn't have a mentor, but
-	I'm interested in that project?"
+question: "A showcased project on the website doesn't have a mentor, but I'm interested in that project?"
```
Comment (jayvdb) [inline _faq/is-mentor-necessary.md]: no , please do as requested in the last review.

(your change makes it go over the preferred line length)

File: _projects/extend-linter-integration.md [hunk 5]
```diff
@@ -80,8 +80,8 @@ challenging task. Hit and trial methods or the usage of a debugger are the
 only possible ways to overcome these errors.
 ```
 E           AssertionError: Lists differ: [&lt;Result object(id=0xce0383d576a84f34848d66d0ddc87479, origin=[632 chars]
-		    710&gt;] != [&lt;Result object(id=0x7e58527926a345f8a2bb36305c58f9f6, origin=[632 chars]400&gt;]
-E           
+            710&gt;] != [&lt;Result object(id=0x7e58527926a345f8a2bb36305c58f9f6, origin=[632 chars]400&gt;]
```
Comment (jayvdb) [inline _projects/extend-linter-integration.md]: this needs to be indented

File: _projects/extend-linter-integration.md (hunk 1 above)
Comment (jayvdb) [reply _projects/extend-linter-integration.md]: this hasnt been done yet

File: _projects/extend-linter-integration.md (hunk 2 above)
Comment (jayvdb) [reply _projects/extend-linter-integration.md]: why isnt this done yet?  :P

File: _faq/gsoc-requirements.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ question: "What are the requirements to be accepted for GSoC?"
...
 
 * You have completed the [coala Newcomers guide](https://coala.io/newcomer)
-* You have a submission to [our usability survey](https://coala.io/usability)
-	so we know that you know how to *use coala*.
-	
\ No newline at end of file
+* You have a submission to [our usability survey](https://coala.io/usability) 
+    you know how to *use coala*.
+    so we know that.
+
```
Comment (jayvdb) [inline _faq/gsoc-requirements.md]: please remove this extra trailing blank line.

File: _faq/gsoc-requirements.md [hunk 7]
```diff
@@ -4,6 +4,7 @@ question: "What are the requirements to be accepted for GSoC?"
 Other than the [requirements from Google](https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation) you will have to meet the following:
 
 * You have completed the [coala Newcomers guide](https://coala.io/newcomer)
-* You have a submission to [our usability survey](https://coala.io/usability)
-	so we know that you know how to *use coala*.
-	
\ No newline at end of file
+* You have a submission to [our usability survey](https://coala.io/usability) 
+    you know how to *use coala*.
```
Comment (jayvdb) [inline _faq/gsoc-requirements.md]: you have changed the words here.  and the first word on this line should be under "You" on the previous line.

File: _faq/is-mentor-necessary.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 question: "A showcased project on the website doesn't have a mentor, but
-	I'm interested in that project?"
+I'm interested in that project?"
```
Comment (jayvdb) [inline _faq/is-mentor-necessary.md]: Why is this here?

See previous comment https://github.com/coala/projects/pull/678#discussion_r228020574

File: _faq/gsoc-requirements.md [hunk 9]
```diff
@@ -5,5 +5,5 @@ Other than the [requirements from Google](https://developers.google.com/open-sou
 
 * You have completed the [coala Newcomers guide](https://coala.io/newcomer)
 * You have a submission to [our usability survey](https://coala.io/usability)
-	so we know that you know how to *use coala*.
-	
\ No newline at end of file
+    You know how to *use coala*.
```
Comment (jayvdb) [inline _faq/gsoc-requirements.md]: Why are you changing the content of this file ?

File: _projects/extend-linter-integration.md [hunk 10]
```diff
@@ -93,12 +93,12 @@ difficult to debug things like:
 E           AssertionError: 0 == 0 : The local bear 'CheckstyleBear' yields no result although it should.
 E           Running bear CheckstyleBear...
 E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/
-			.local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
-			CheckstyleGood.java'
+              .local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
```
Comment (jayvdb) [inline _projects/extend-linter-integration.md]: four space indent, not two.

File: _projects/extend-linter-integration.md [hunk 11]
```diff
@@ -93,12 +93,12 @@ difficult to debug things like:
...
+            CheckstyleGood.java'
```
Comment (jayvdb) [inline _projects/extend-linter-integration.md]: four space indent

File: _faq/is-mentor-necessary.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 ---
 question: "A showcased project on the website doesn't have a mentor, but
-	I'm interested in that project?"
+             I'm interested in that project?"
```
Comment (jayvdb) [inline _faq/is-mentor-necessary.md]: "I'm .." on this line should be aligned exactly under "A showcase.." on the previous line.

i.e. remove two spaces.

File: _faq/gsoc-requirements.md (hunk 9 above)
Comment (jayvdb) [reply _faq/gsoc-requirements.md]: This hasnt been modified since the last review.

File: _faq/gsoc-requirements.md [hunk 13]
```diff
@@ -5,5 +5,5 @@ Other than the [requirements from Google](https://developers.google.com/open-sou
 
 * You have completed the [coala Newcomers guide](https://coala.io/newcomer)
 * You have a submission to [our usability survey](https://coala.io/usability)
-	so we know that you know how to *use coala*.
-	
\ No newline at end of file
+    so we know that you know how to *use coala*.
```
[the remark this answers is not in the document]
Comment (jayvdb) [reply _faq/gsoc-requirements.md]: two spaces need to be removed.

File: _faq/is-mentor-necessary.md (hunk 12 above)
Comment (jayvdb) [reply _faq/is-mentor-necessary.md]: Still two spaces need to be removed

File: _projects/extend-linter-integration.md (hunk 11 above)
Comment (jayvdb) [reply _projects/extend-linter-integration.md]: still not done.  Read this page: https://github.com/coala/projects/pull/678/files to find the context of my comment. Then it makes sense!

**Comment by SerekKiri** [comment]
@ishanSrt We decide on gitter that here it will be two commits

File: _projects/extend-linter-integration.md (hunk 10 above)
Comment (jayvdb) [reply _projects/extend-linter-integration.md]: Not modified yet.  You need to add two more spaces here.

File: _projects/extend-linter-integration.md (hunk 11 above)
Comment (jayvdb) [reply _projects/extend-linter-integration.md]: Still not modified.

**Comment by ishanSrt** [comment]
Ok my bad 😅 then please change the body of the most recent commit

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/678/commits/346464ac1cc1df4a6e5fcb5960f2209a124e314b https://github.com/coala/projects/pull/678/commits/bd50ee6b4927c4ef6a9f478056074783ab266b57

**Comment by jayvdb** [comment]
@gitmate-bot ff
